src/content/guides/getting-started.md

```diff
@@ -178,25 +179,25 @@ __dist/index.html__
 
 In this setup, `index.js` explicitly requires `lodash` to be present, and binds it as `_` (no global scope pollution). By stating what dependencies a module needs, webpack can use this information to build a dependency graph. It then uses the graph to generate an optimized bundle where scripts will be executed in the correct order.
 
-With that said, let's run `npx webpack` with our script as the [entry point](/concepts/entry-points) and `bundle.js` as the [output](/concepts/output). The `npx` command, which ships with Node 8.2 or higher, runs the webpack binary (`./node_modules/.bin/webpack`) of the webpack package we installed in the beginning:
+With that said, let's run `npx webpack` with no arguments, which will take our script at `src/index.js` as the [entry point](/concepts/entry-points), and will generate `dest/main.js` as the [output](/concepts/output).. The `npx` command, which ships with Node 8.2 or higher, runs the webpack binary (`./node_modules/.bin/webpack`) of the webpack package we installed in the beginning:
```

Comment:
Looks good! One small thing, can you please remove `with no arguments` as it's already clear what we're running. Also, I guess it's `dist/main.js` based on config in the example above. Please remove unneeded period after `output` link.

Reply:
Oops! Will do that now.